src/main/kotlin/io/sip3/twig/ce/service/SearchService.kt

```diff
@@ -74,6 +82,7 @@ abstract class SearchService {
         val name = attribute.substringAfter("sip.")
                 .substringAfter("ip.")
                 .substringAfter("rtp.")
+                .substringAfter("rtcp.")
                 // Accordingly to SIP3 conventions attributes have names like r-factor, setup-time, e.t.c
                 // However, in MongoDB all fields have underscore instead - `r_factor`, `setup_time`, e.t.c
                 .replace("-", "_")
```

Comment:
Let's remove this line. As far as I can see we don't have `r-factor` or `setup-time` notation anymore